config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'categories', to: 'categories#index'
```

Comment:
en este caso sería mejor usar:
```suggestion
  resources :categories, only: %i[index]
```